addons/addon-base-raas/packages/base-raas-services/lib/data-source/data-source-bucket-service.js

```diff
@@ -103,11 +103,6 @@ class DataSourceBucketService extends Service {
         true,
       );
 
-    // SSE (server side encryption) using S3 and not KMS is not supported
-    if (rawBucketEntity.sse === 's3') {
-      throw this.boom.notSupported('SSE S3 is not supported', true);
-    }
```

Comment:
When set to `s3`, could you confirm it you able to write new data through a workspace instance, and read it successfully?

Reply:
Already did that as part of Windows and Linux testing